azure-pipelines.yml

```diff
@@ -0,0 +1,70 @@
+name: 3.0.0-ci.$(Date:yyyyMMdd).$(Rev:r) # equal to $(Build.BuildNumber)
+trigger:
+  branches:
+    include:
+    - dev
+    - master
+    - hotfix/*
+    - release/*
```

Comment:
Would it work for commits in pull requests?

Reply:
_Sorry for the delay, just circling back to this today._

Yes, I believe this would include any PRs against those branches. Per [the docs](https://docs.microsoft.com/azure/devops/pipelines/build/ci-build-git?view=azure-devops&tabs=yaml#validate-pull-requests):

> Unless you specify `pr` triggers in your YAML file, pull request builds are automatically enabled for all branches.